smartmin/views.py

```diff
@@ -420,6 +420,8 @@ class SmartTemplateView(SmartView, TemplateView):
 
 
 class SmartReadView(SmartView, DetailView):
+    slug_field = None
+    slug_url_kwarg = None
```

Comment:
No I mean using only  slug_url_kwarg, I do not see slug_field being used; is it a django config?